### PR TITLE
[#135592] Add an outside contact info

### PR DESCRIPTION
**Pull request by noelrappin**
To the creation of a new purchase order. Everything was put in the c2po engine.

https://pm.tablexi.com/issues/135592

### CODE REVIEW

File: vendor/engines/c2po/app/views/facility_accounts/account_fields/_purchase_order_account.html.haml [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
 = f.input :description, required: true
 
 = render partial: 'facility_accounts/account_fields/affiliate', locals: { f: f }
+
 = render partial: 'facility_accounts/account_fields/remittance_information', locals: { f: f }
 
+= f.input :outside_contact_info,
+  label: text("outside_contact_label"),
+  hint: text("outside_contact_hint"),
+  hint_html: { :class => 'help-inline' }
```
Comment (meara) [inline vendor/engines/c2po/app/views/facility_accounts/account_fields/_purchase_order_account.html.haml]: `haml-lint`, pretty please. 😁

Comment (noelrappin) [inline vendor/engines/c2po/app/views/facility_accounts/account_fields/_purchase_order_account.html.haml]: Done in the other PR

Comment (meara) [inline vendor/engines/c2po/app/views/facility_accounts/account_fields/_purchase_order_account.html.haml]: Ohhhh, I see what you did now. Took me a second to figure out your branches. 👍

File: db/schema.rb [hunk 2]
```diff
@@ -43,6 +43,7 @@
     t.integer  "facility_id",            limit: 4
     t.integer  "affiliate_id",           limit: 4
     t.string   "affiliate_other",        limit: 255
+    t.string   "outside_contact_info",   limit: 255
```
Comment (jhanggi) [inline db/schema.rb]: I suspect whichever of these PRs goes in last will need to get updated to deal with merge conflicts.

File: vendor/engines/c2po/app/views/facility_accounts/account_fields/_purchase_order_account.html.haml [hunk 3]
```diff
@@ -1,7 +1,16 @@
-= f.input :account_number, label: t('facility_accounts.purchase_orders.account_number'), required: true
-= f.input :formatted_expires_at, required: true, input_html: { class: 'datepicker' }, label: Account.human_attribute_name(:expires_at)
+= f.input :account_number, label: text("account_number"), required: true
+
+= f.input :formatted_expires_at, required: true,
+  input_html: { class: 'datepicker' },
```
Comment (jhanggi) [inline vendor/engines/c2po/app/views/facility_accounts/account_fields/_purchase_order_account.html.haml]: Use double quotes. Looks like that didn't get picked up in the other PR.

**Pull request merged**